From 2e1c40afd9f861fd9878534293134e12e8a59239 Mon Sep 17 00:00:00 2001
From: Hou Qi <qi.hou@nxp.com>
Date: Thu, 2 Nov 2023 16:21:06 +0900
Subject: [PATCH 17/19] V4L2VDA: Enlarge input buffer count to 16

Some stream can decode one frame only after queuing over 8 buffers.
So enlarge input buffer count to 16 to avoid such stream cannot play.

Upstream-Status: Inappropriate [NXP specific]
---
 media/gpu/v4l2/legacy/v4l2_video_decode_accelerator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/gpu/v4l2/legacy/v4l2_video_decode_accelerator.h b/media/gpu/v4l2/legacy/v4l2_video_decode_accelerator.h
index d2448871485f9..1b205abaafdda 100644
--- a/media/gpu/v4l2/legacy/v4l2_video_decode_accelerator.h
+++ b/media/gpu/v4l2/legacy/v4l2_video_decode_accelerator.h
@@ -157,7 +157,7 @@ class MEDIA_GPU_EXPORT V4L2VideoDecodeAccelerator
  private:
   // These are rather subjectively tuned.
   enum {
-    kInputBufferCount = 8,
+    kInputBufferCount = 16,
     // TODO(posciak): determine input buffer size based on level limits.
     // See http://crbug.com/255116.
     // Input bitstream buffer size for up to 1080p streams.
-- 
2.34.1

